decidim-admin/app/commands/decidim/admin/create_scope.rb

```diff
@@ -31,7 +32,11 @@ def call
       def create_scope
         Scope.create!(
           name: form.name,
-          organization: form.organization
+          organization: form.organization,
+          code: form.code,
+          scope_type_id: form.scope_type_id,
```

Comment:
Also: I think Scope Types should be optional. Eases the transition from our current data model, and makes sense when having arbitrary trees of scopes in scenarios where they don't have a clear categorization.